source/configure/manage-user-surveys.rst

```diff
@@ -0,0 +1,79 @@
+Manage user surveys
+===================
+
+.. include:: ../_static/badges/allplans-selfhosted.rst
+  :start-after: :nosearch:
+
+Use the Mattermost User Survey integration to gather direct feedback from your Mattermost users to identify what's working well and what's not with your Mattermost instance. All user responses are stored in and remain within your self-hosted deployment. You can export a CSV report of NPS scores and user responses for further analysis or to share with Mattermost.
+
+You can schedule when each survey begins, define how long each survey lasts, specify teams excluded from the survey, as well as customize both a welcome message and a user question you want feedback on.
+
+Setup
+------
+
+You must be a Mattermost system admin to `upload the plugin <#upload>` to your Mattermost self-hosted deployment `enable it <#enable>`__,  `create surveys <#create-surveys>`__, and `export survey responses <#export-survey-responses>` using the System Console.
+
+The User Survey integration is compatible with the following :doc:`Mattermost Server versions </deploy/mattermost-changelog>`:
+
+  - v9.6 or later
+  - v9.5.2+ (Extended Support Release - ESR)
+  - v9.4.4+
+  - v9.3.3+
+  - v8.1.11+ (Extended Support Release - ESR)
+
+Upload
+------
+
+The User Survey integration must be installed using the `latest binary available for download from the plugin repository <https://github.com/mattermost/mattermost-plugin-user-survey/releases>`_
```

Comment:
@harshilsharma63 - Please confirm whether this is true (or whether the plugin will be available in the in-product/external Marketplace).

Reply:
This is correct, it is currently not available in the marketplace